src/index.d.ts

```diff
@@ -333,6 +333,19 @@ declare namespace electronPackager {
      * @category macOS
      */
     extendInfo?: string | { [property: string]: any }; // eslint-disable-line @typescript-eslint/no-explicit-any
+    /**
+     * When the value is a string, specifies the filename of a `plist` file. Its contents are merged
+     * into all the Helper app's `Info.plist` files.
```

Comment:
Nit(grammar): pretty sure it's a plural possessive because it's multiple helper apps.

```suggestion
     * into all the Helper apps' `Info.plist` files.
```